src/sss_client/common_private.h

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
I know this is a nitpick, but it would be nice to add the license block and ifdef guards as we add to other headers as well.

Reply:
fixed